test/parallel/test-https-connect-address-family.js

```diff
@@ -5,29 +5,42 @@ if (!common.hasCrypto) {
   return;
 }
 
-const assert = require('assert');
-const https = require('https');
-
 if (!common.hasIPv6) {
   common.skip('no IPv6 support');
   return;
 }
 
-const ciphers = 'AECDH-NULL-SHA';
-https.createServer({ ciphers }, function(req, res) {
-  this.close();
-  res.end();
-}).listen(common.PORT, '::1', function() {
-  const options = {
-    host: 'localhost',
-    port: common.PORT,
-    family: 6,
-    ciphers: ciphers,
-    rejectUnauthorized: false,
-  };
-  // Will fail with ECONNREFUSED if the address family is not honored.
-  https.get(options, common.mustCall(function() {
-    assert.strictEqual('::1', this.socket.remoteAddress);
-    this.destroy();
+const assert = require('assert');
+const https = require('https');
+const dns = require('dns');
+
+function runTest() {
+  const ciphers = 'AECDH-NULL-SHA';
+  https.createServer({ ciphers }, common.mustCall(function(req, res) {
+    this.close();
+    res.end();
+  })).listen(common.PORT, '::1', common.mustCall(function() {
+    const options = {
+      host: 'localhost',
+      port: common.PORT,
+      family: 6,
+      ciphers: ciphers,
+      rejectUnauthorized: false,
+    };
+    // Will fail with ECONNREFUSED if the address family is not honored.
+    https.get(options, common.mustCall(function() {
+      assert.strictEqual('::1', this.socket.remoteAddress);
+      this.destroy();
+    }));
   }));
+}
+
+dns.lookup('localhost', {family: 6, all: true}, (err, addresses) => {
+  if (err)
+    throw err;
+
+  if (addresses.some((val) => { return val.address === '::1'; }))
```

Comment:
I thought I was going with the de facto standard in the project code as it exists, but it looks like you are correct. There are over 150 instances where the braces are already omitted and only around 20 cases where there are braces included even though they can be removed.

So, yeah, I'll remove these.